app/models/forms/risks/step_manager.rb

```diff
@@ -0,0 +1,77 @@
+module Forms
+  module Risks
+    class StepManager
+      RISKS_STEPS = DoubleLinkedList.new.tap do |l|
+        l << Forms::Risks::ToSelf
+        l << Forms::Risks::FromOthers
+        l << Forms::Risks::Violence
+        l << Forms::Risks::Harassments
+        l << Forms::Risks::SexOffences
+        l << Forms::Risks::NonAssociationMarkers
+        l << Forms::Risks::Security
+        l << Forms::Risks::SubstanceMisuse
+        l << Forms::Risks::ConcealedWeapons
+        l << Forms::Risks::Arson
+        l << Forms::Risks::Communication
+      end
+
+      def self.build_step_for(step_name, escort)
+        node = RISKS_STEPS.find { |n| n.value.name == step_name }
+        form = node.value.new(escort.risks)
+        Step.new(node, form, escort)
+      end
+
+      def self.total_steps
+        RISKS_STEPS.count
+      end
+    end
+
+    class Step
```

Comment:
This is just duped from last weeks healthcare wizard right? We need to condense this stuff into a single approach, as far as I can see healthcare & risks have the same behaviour.